baldrick/plugins/github_pull_requests.py

```diff
@@ -105,6 +106,11 @@ def process_pull_request(repository, number, installation):
         if result is not None:
             results.update(result)
 
+    # Special message for a special day (hacky hack hack)
+    not_boring = pr_handler.get_config_value('not_boring', cfg_default=True)
+    if not_boring and is_special_day_now():
+        pr_handler.submit_comment('')
```

Comment:
Well that's a pretty boring comment!

Reply:
Ah wait I understand, ignore me. However, could we move the whole logic from ``submit_comment`` to here? I ran into an issue in #69 which happens when submit_comment is called outside of the context of the flask app (which it is for the stale scripts) and it would be easier to just have all the logic in this file rather than always adding it to all comments.

Reply:
Hmm I'll see to it when I get a chance.